tests/cases/standalone/common/tql-explain-analyze/analyze.result

```diff
@@ -59,7 +59,7 @@ TQL ANALYZE (0, 10, '1s', '2s') test;
 |_|_|_CoalesceBatchesExec: target_batch_size=8192 REDACTED
 |_|_|_FilterExec: j@1 >= -2000 AND j@1 <= 12000 REDACTED
 |_|_|_RepartitionExec: partitioning=REDACTED
-|_|_|_SinglePartitionScanner: <SendableRecordBatchStream> REDACTED
+|_|_|_SeqScan: [ScanPart(1 memtables, 0 file ranges, time range: (1::Millisecond, 2::Millisecond))] REDACTED
```

Comment:
can you put `time range` into `VERBOSE` mode? they are too "verbose"

Reply:
😠